Public/Src/FrontEnd/Nuget/NugetAnalyzedPackage.cs

```diff
@@ -532,44 +371,38 @@ private bool TryParseDependenciesFromNuSpec()
             {
                 if (group.Attribute("targetFramework") != null && NugetFrameworkMonikers.TargetFrameworkNameToMoniker.TryGetValue(group.Attribute("targetFramework").Value, out Moniker targetFramework))
                 {
-                    // If there is at least one valid dependency for a known framework, then the package is defined as managed
-                    IsManagedPackage = group.Elements().Any();
-                    TargetFrameworkWithFallbacks.Add(targetFramework);
-
-                    // TODO:Nuget: Clean this up once we got end to end TFM support inside our Nuget toolchain, this is needed to not break backwards compatibility
-                    if (string.IsNullOrEmpty(this.Tfm) &&
-                        (targetFramework.Equals(NugetFrameworkMonikers.NetCore) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard10) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard11) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard12) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard13) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard14) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard15) ||
-                        targetFramework.Equals(NugetFrameworkMonikers.NetStandard16)))
+                    if (group.Elements().Any())
                     {
-                        continue;
-                    }
+                        // If there is at least one valid dependency for a known framework, then the package is defined as managed
+                        IsManagedPackage = true;
 
-                    // If the package has a pinned tfm and the groups tfm does not match, skip the groups dependency resolution
-                    if (!string.IsNullOrEmpty(this.Tfm) && NugetFrameworkMonikers.TargetFrameworkNameToMoniker.TryGetValue(this.Tfm, out Moniker pinnedTfm) && !PathAtom.Equals(pinnedTfm, targetFramework))
-                    {
-                        continue;
-                    }
+                        // Only add the group dependency target framework if the nuget package itself also contains specific assemblies of the same version

```

Comment:
> group dependency t [](start = 39, length = 19)

Thanks, this is what Nora ran into with her PR:
[https://github.com/microsoft/BuildXL/pull/259/files](https://github.com/microsoft/BuildXL/pull/259/files) file `BuildXL.Utilities.dsc`